### PR TITLE
ipset: support creation of inet6 sets

### PR DESCRIPTION
**Pull request by stv0g**
This PR adds support for AF_INET6 address and network management in the ipset code.

### CODE REVIEW

File: ipset_linux.go [hunk 1]
```diff
@@ -401,9 +414,11 @@ func (result *IPSetResult) parseAttrData(data []byte) {
 		case nl.IPSET_ATTR_IP | nl.NLA_F_NESTED:
 			for nested := range nl.ParseAttributes(attr.Value) {
 				switch nested.Type {
-				case nl.IPSET_ATTR_IP | nl.NLA_F_NET_BYTEORDER:
+				case nl.IPSET_ATTR_IPV4 | nl.NLA_F_NET_BYTEORDER: // fallthrough
+				case nl.IPSET_ATTR_IPV6 | nl.NLA_F_NET_BYTEORDER:
 					result.Entries = append(result.Entries, IPSetEntry{IP: nested.Value})
-				case nl.IPSET_ATTR_IP:
+				case nl.IPSET_ATTR_IPV4: // fallthrough
```
Comment (vishvananda) [inline ipset_linux.go:420]: if the goal is for these cases to fallthrough, in go they need an explicit fallthrough statement: https://golang.org/ref/spec#Switch_statements

**Comment by vishvananda** [comment]
there are multiple mistakes in fallthrough in this pr. Please fix and ammedn the commit. Thanks!

File: ipset_linux.go (hunk 1 above)
Comment (stv0g) [reply ipset_linux.go]: Oh, I really dont remember what I was thinking here..

Should be fixed now.

**Comment by vishvananda** [comment]
Looks like the tests are failing in CI. Does it need a newer kernel version? or a certain kernel module?

**Comment by aboch** [comment]
@stv0g

**Comment by aboch** [comment]
@stv0g

**Comment by stv0g** [comment]
Sorry, I dont have the bandwidth anymore to work on this.

**Pull request closed without merging**